docs/Installation.md

```diff
@@ -41,34 +40,55 @@ minutes to complete.
 Building and flashing the micro-controller
 ==========================================
 
+Via Octoprint, make sure that there is no active serial connection to the printer.
+Fom the OctoPrint web page, under the "Connection" section, click "Disconnect"
+
 To compile the micro-controller code, start by running these commands
-on the Raspberry Pi:
+on the Raspberry Pi via an SSH command line session:
 
 ```
 cd ~/klipper/
 make menuconfig
 ```
 
-Select the appropriate micro-controller and review any other options
-provided. Once configured, run:
+Manually select the appropriate micro-controller, processor speed (mhz), and baurdrate. 
+Review any other options provided. After choosing the correct settings for your printers
+micro-controller, you will exit the configuration window by choosing 'Exit'. Then select
+'Yes' to save your new configuration.  
```

Comment:
I don't think explaining how "Exit" and "Yes" works is necessary.

Reply:
Then explain how to exit the program and save it without recognizing that the profile needs to be saved for the compiler to have something to work with.

It's not going to save itself.

Reply:
The Save and Exit functions have been removed. All that you need to do now is select exit. What I'm saying is that explaining that you need to "exit the configuration window by choosing 'Exit'. Then select 'Yes' to save your new configuration." is not necessary, as it's self-explanatory